gcp/api/server.py

```diff
@@ -429,25 +432,24 @@ def query_by_commit(context: grpc.ServicerContext,
   """Query by commit."""
   query = osv.AffectedCommits.query(osv.AffectedCommits.commits == commit)
   bug_ids = []
-  it = query.iter()
+  it: ndb.QueryIterator = query.iter(keys_only=True)
   gsd_count = 0
 
   while (yield it.has_next_async()):
-    affected_commits = it.next()
-    # Avoid requiring a separate index to include this in the initial Datastore
-    # query. The number of these should be very little to just iterate through.
-    if not affected_commits.public:
-      continue
+    # Affect commits key follows this format:
+    # <BugID>-<PageNumber>
+    affected_commit_key: ndb.Key = it.next()
+    bug_id: str = affected_commit_key.id().rsplit("-", 1)[0]
```

Comment:
It's hard to get an estimate without just making the index,, but I'm guessing the index created will be huge because:
> Only indexed properties can be projected.
> This means that all properties used in a query (projected or filters) must exist in the same index. So, select tag from Task where priority = 1 requires a composite index on priority then tag.

Since we are filtering by commit_hash, we will have to hash that column, which will result in roughly another terabyte of indexes.